src/dropdown/dropdown.js

```diff
@@ -187,17 +187,30 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
     }
   };
 
+  function toggleDropdownUp() {
```

Comment:
Thanks, I'm going back to the drawing board and will let you know if I changed the position service.